### PR TITLE
adding Versions section to readme

### PR DESCRIPTION
**Pull request by SomkaPe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@
 
 IdentityModel Extensions for .Net provide assemblies that are interesting for web developers that wish to use federated identity providers for establishing the callers identity. 
 
+## Versions
+Current version - 5.0
```
Comment (polita) [inline README.md]: 5.0.0

**Comment by SomkaPe** [comment]
@brentschmaltz  please provide release branches where we should push this change

**Comment by polita** [comment]
@SomkaPe It should go into master, dev, and release4x.

File: README.md (hunk 1 above)
Comment (SomkaPe) [inline README.md]: 5.0 changed to 5.0.0

**Comment by polita** [comment]
:shipit:

**Comment by brentschmaltz** [comment]
:shipit:

**Pull request merged**